src/PresignUrlMiddleware.php

```diff
@@ -114,7 +114,7 @@ private function createPresignedUrl(
         if (!empty($this->extraQueryParams[$cmdName])) {
             foreach ($this->extraQueryParams[$cmdName] as $param) {
                 if (!strpos($currentQueryParams, $param)) {
-                    $paramsToAdd =  "&{$param}={$cmd[$param]}";
+                    $paramsToAdd =  "&{$param}=".urlencode($cmd[$param]);
```

Comment:
nit: just needs a space on either side of the '.' for clarity - otherwise looks good to me